_security-analytics/usage/findings.md

```diff
@@ -37,12 +37,31 @@ As one more alternative, you can select an option from the **Recently used date
+The **Findings** list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.
 <br><img src="{{site.url}}{{site.baseurl}}/images/Security/finding-list.png" alt="A list of all findings" width="85%">
+
 Use the **Rule severity** dropdown list to filter the list of findings by severity. Use the **log type** dropdown list to filter the list by log type.
-Each finding in the list includes a finding ID. You can select the ID to open the Finding details pane, which describes the finding by parameters defined when creating the detector.
-The Actions column includes two options for each finding:
-* The diagonal arrow provides another way to open the Findings detail pane.
-* The bell icon allows you to open the Create detector alert trigger pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
+
+The **Actions** column includes two options for each finding:
+* The diagonal arrow provides a way to open the **Finding details** pane, which describes the finding by parameters defined when creating the detector and includes the document that generated the finding.
+* The bell icon allows you to open the **Create detector alert trigger** pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
 For details on setting up an alert, see [Set up alerts]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-3-set-up-alerts) in detector creation documentation.
 
+Each finding in the list also includes a **Finding ID**. You can select the ID as another option for opening the **Finding details** pane. An example of **Finding details** is shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/findings1.png" alt="Finding details pane" width="60%">
+
+### Viewing surrounding documents
+
+The **Finding details** pane contains specific information about the finding, including the document that generated the finding. To investigate the series of events that led to the finding or followed the finding, you can select **View surrounding documents** to open the document in the **Discover** panel and view other documents preceding or following it.
+
+1. Open **Finding details** by selecting the **Finding ID** in the list of findings.
+1. In the **Documents** section, select **View surrounding documents**. If an index pattern already exists for the document, the **Discover** panel opens and displays the document. If an index pattern does not exist, the **Create index pattern to view documents** window opens and prompts you to create an index pattern, as shown in the following image.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/findings2.png" alt="popup window prompting users to create an index pattern" width="60%">
```

Comment:
Remove `<br>` and add a new line before and after the image.

Reply:
These images are in steps. If I add spaces it breaks the sequence and doesn't render the step numbers correctly. Maybe you're suggesting there's a hard break that leaves the steps intact. Will look into it.
Update: looked and found. Should be all set.

Reply:
If you need an image to be under a step, tab it over one tab. The tabbing makes the list intact.